src/beman/optional26/tests/optional.t.cpp

```diff
@@ -596,6 +596,21 @@ TEST(OptionalTest, RangeTest) {
     }
 }
 
+TEST(OptionalTest, HashTest) {
```

Comment:
Maybe we should move this test at the end of this file? I'm not expecting to see hashing tests before tests for contructors or other basic features.

Reply:
It's before the tests that I imported from views maybe, but since that doesn't really make sense for anyone but me, I'll rename those tests and slot the hash in later.